Maple2.File.IO/Nif/NiPhysXPropDesc.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace Maple2.File.IO.Nif;
+
+public class NiPhysXPropDesc : NifBlock {
+    public class StateString {
+        public string String;
+        public uint Value;
+
+        public StateString(string key, uint value) {
+            String = key;
+            Value = value;
+        }
+    }
+
+    public class State {
```

Comment:
A type to hold a single type seems redundant, why not just use `List<StateString>` directly?